docs/snippets/ov_python_exclusives.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+#! [auto_compilation]
+import openvino.runtime as ov
+
+compiled_model = ov.compile_model("model.xml")
+#! [auto_compilation]
+
+#! [properties_example]
+core = ov.Core()
+
+input_a = ov.opset8.parameter([8])
+res = ov.opset8.relu(input_a)
+model = ov.Model(res, [input_a])
+compiled = core.compile_model(model, "CPU")
+
+print(model.inputs)
+print(model.outputs)
+
+print(compiled.inputs)
+print(compiled.outputs)
+#! [properties_example]
+
+#! [tensor_basics]
+data_float64 = np.ones(shape=(2,8))
+
+t = ov.Tensor(data_float64)
```

Comment:
may be rename `t` to `tensor`?
I don't like that in public docs one-letter variable is used

Reply:
Done!